.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<!-- 
+
+Read the Lab Website Template documentation BEFORE opening this pull request:
```

Comment:
Since folks clone this, should it refer to their own repo? I'm not sure how to make a relative link in a PR template though...

Reply:
I want to refer to the official LWT docs which explain how to make basic edits. Website owners can change the pull request template to have their own guidelines if they want.

But you raise an interesting question. Does the wiki get copied over too when you fork/template a repo? I kinda hope not, I don't want different versions of the docs floating around...